src/card/card.jsx

```diff
@@ -21,6 +21,13 @@ const Card = React.createClass({
     expandable: React.PropTypes.bool,
 
     /**
+     * Whether this card is expanded.
+     * If `true` or `false` the component is controlled.
+     * if `null` the component is uncontrolled.
+     */
+    expanded: React.PropTypes.bool,
```

Comment:
The name of the property makes me think about an older discussion we had with `open` vs `opened` for our `<LeftNav />, `<Dialog />`, `<Snackbar />`, `<IconMenu />` and <Popover />`. We choose `open`.
What do you think about calling this property `expand`? That's more active than passive. cc @alitaheri @newoga  @mbrookes